libwild/src/elf_writer.rs

```diff
@@ -221,6 +222,21 @@ fn write_file_contents<A: Arch>(sized_output: &mut SizedOutput, layout: &Layout)
             Ok(())
         })?;
 
+    let sframe_buffer = section_buffers.get_mut(output_section_id::SFRAME);
+    if !sframe_buffer.is_empty() {
+        // TODO: To reliably ensure `sort_sframe_section` is called for all `.sframe` sections, you
```

Comment:
This sounds a bit like what we currently do for `.eh_frame_hdr`. https://github.com/davidlattimore/wild/blob/1b15e1daacf414d5501b22c5154cbf1a88871849/libwild/src/elf_writer.rs#L138
Perhaps we can call this function at the same point?